test-files/awscfn/RDS_MySQL_With_Read_Replica.yaml

```diff
@@ -0,0 +1,125 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
Similar to the Azure and GCP TF files, if we don't use these, can you please remove them from the repo (we can always re-add them if required for a workflow)?